tests/conftest.py

```diff
@@ -92,8 +92,23 @@ def repository_paths(tmp_path_factory):
     create_repositories(base_path)
 
     repositories = {
-        "high_entropy": base_path / "high_entropy",
-        "low_entropy": base_path / "low_entropy",
+        "test_repo_1": base_path / "test_repo_1",
+        "test_repo_2": base_path / "test_repo_2",
     }
 
     yield repositories
+
+
+@pytest.fixture
+def test_file_sets():
```

Comment:
Consider adding to the name here about "which" file set (repo?). Also, consider avoiding the prefix `test_` to help avoid this being interpreted as a test vs fixture.